<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>-1</y>
      <width>980</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,255);</string>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>831</width>
       <height>591</height>
      </rect>
     </property>
     <widget class="QWidget" name="record_page">
      <widget class="QTableWidget" name="log_table">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>40</y>
         <width>760</width>
         <height>420</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>125</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>房间号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>身份证号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>请求温度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>当前温度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>风速</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>能量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>金额</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>480</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="text">
        <string>打印日志</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="adduser_page">
      <widget class="QLineEdit" name="room_id_lineEdit">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>220</y>
         <width>250</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>身份证号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="card_id_lineEdit">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>140</y>
         <width>250</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>220</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>房间号</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add_pushButton">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>310</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="deleteuser_page">
      <widget class="QLineEdit" name="room_id_lineEdit2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>200</y>
         <width>250</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>200</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>房间号</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_pushButton">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>310</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="changemodel">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>160</y>
         <width>131</width>
         <height>40</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>当前模式</string>
       </property>
      </widget>
      <widget class="QLabel" name="model_label">
       <property name="geometry">
        <rect>
         <x>299</x>
         <y>160</y>
         <width>201</width>
         <height>40</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="changemodel_Button">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>250</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="seerecord_Button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>151</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>查看状态</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changevalue_Button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>修改模式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>982</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>主机控制系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteuser_Button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>删除房客</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adduser_Button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加房客</string>
     </property>
    </widget>
    <zorder>stackedWidget</zorder>
    <zorder>seerecord_Button</zorder>
    <zorder>changevalue_Button</zorder>
    <zorder>deleteuser_Button</zorder>
    <zorder>adduser_Button</zorder>
    <zorder>label</zorder>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/login.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="logout_Button">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>0</y>
       <width>151</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注销</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>widget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
